header.php

```diff
@@ -48,9 +48,6 @@
 			} else {
 			// Show Header Image, Site Title, and Site Tagline on everything except Single Pages
 				independent_publisher_site_info();
-				if ( get_option( 'iw_single_author' ) ) {
```

Comment:
@dshanske Confused about why code, that was added in adc2dd24c275c5a2b3b2e3f4423cc36cc60bd23d, was removed in this commit. There doesn't seem to be any associated commit message. Was the removal of this code a mistake?

Reply:
I can't remember. It was a while ago I submitted the PR. It may have been I moved the logic elsewhere.